project/src/main/scala/edu/gemini/osgi/tools/idea/IdeaProject.scala

```diff
@@ -318,19 +319,20 @@ def project: xml.Elem =
 
   private def libraryTable: xml.Elem =
     <component name="libraryTable">
-      <library name="scala-compiler">
-        <CLASSES>
-          {scalaInstance.allJars.map(jarUrl)}
-        </CLASSES>
-      </library>
-      <library name="scala-library">
-        <CLASSES>
-          {scalaInstance.allJars.map(jarUrl)}
-        </CLASSES>
+      <library name="scala-sdk" type="Scala">
+        <properties>
+          <option name="languageLevel" value="Scala_2_11" />
```

Comment:
Unclear.  I think the corresponding Java setting controls which language features are supported by the editor.  At any rate, it should probably match the Scala version we're actually using so I updated it to read from `scala.util.Properties.versionNumberString`.